### ISSUE TITLE
Problem with tumblers feed

### ISSUE THREAD

**Issue by azraal** [nt1m/livemarks#195]
**Feed URL**
https://deathclawdesu.tumblr.com/rss
https://wheresmybubble.tumblr.com/rss

**Add-on version**
2.1

**Describe the bug**
The rss adding page says
"Impossible de récupérer le flux."

Opening it in vivaldi browser (chrome based), it's recognized as a feed and opening it edge shows the xml. 

Note: 
My browser is in French I don't know if this is an issue.
My firefox version is 68.0.1

**To Reproduce**
1. Navigate to the url marked in the feed URL section (those are the only two I tested but I think every tumbler can have this rss feed)
2. See the error message.

**Expected behavior**
The feed should be detected and added correctly.

**Comment by nt1m** [comment]
@azraal The feed works on my side.
Does it work if you turn off the Firefox "Content Blocking"/"Tracking protection" feature?

I think in French it's "Blocage de contenu"/"Protection contre le pistage" or something like that.

**Comment by azraal** [comment]
Ok so I tried it yesterday. I also disabled several security addons. it didn't work. In the console I had a "no xml found" in suscribe.js if I remember well. So I put everything back as it was. This morning it seems to work.

So maybe it was something in my config, an issue or an old tweak in my config that was reset by messing with the option.

Anyway, thanks for your help. Unless you have questions on what I did exactly we can close the ticket.
